pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Context.java

```diff
@@ -262,6 +264,21 @@
      */
     String getSecret(String secretName);
 
+    /**
+     * Get the pulsar admin client.
+     *
+     * @return The instance of pulsar admin client
+     */
+    PulsarAdmin getPulsarAdmin();
+
+    /**
+     * Get the pulsar admin client by cluster name.
```

Comment:
can we explain here when it is better to use this method instead of the other ?

Reply:
Pulsar recently adds support for multiple clusters, i.e. input topics from cluster A and output topics from cluster B. This method just provides the same support for admin calls.